iam_openapi_spec.yml

```diff
@@ -3504,6 +3551,33 @@ components:
           items:
             $ref: '#/components/schemas/PolicyStatement'
 
+    CreatePolicyFromTemplateRequest:
+      description: Payload to create policy from template
+      type: object
+      additionalProperties: false
+      required:
+        - name
+        - templateName
+        - userId
+      properties:
+        name:
+          type: string
+          maxLength: 50
+        templateName:
+          type: string
+          maxLength: 512
+        userId:
+          type: string
+          maxLength: 512
+        subOrganizationId:
+          type: string
+          maxLength: 50
+        allAccountsAccess:
+          type: boolean
+        accountId:
```

Comment:
Instead of accepting account_id, we must accept a map of key values and pass it to policy template. If any of the variables required by template is missing in request, we must throw an error.

Feel free to ad required_variables as an array to policy template table if you feel it's reqired